app/jobs/event_notification_job.rb

```diff
@@ -0,0 +1,13 @@
+class EventNotificationJob < ApplicationJob
+  def self.perform
+    Event.requiring_notification.each do |event|
```

Comment:
Probably want to eager load presentations and person here so that they are all fetch in the same query. This way we avoid creating an N+1 query to the database. 

If there were a large number of events involved, we might want to consider using the `find_each` AR method rather than a simple each. but for now its probably fine